src/index.html

```diff
@@ -3,11 +3,391 @@
+                  type="email"
+                  name="email"
+                  placeholder="E-mail"
+                  id="email"
+                  required
+                ></input>
+
+                <button
+                  class="subscription__arrow-button"
+                  type="submit"
+                ></button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__inf">
+          <div class="footer__top">
+            <a href="#" >
+              <img
+                class="footer__logo"
+                src="./images/icons/NAMU.svg"
+                alt="NAMU"
+              >
+            </a>
+
+            <div class="footer__icons">
+              <a
+                class="footer__icons-link"
+                href="https://uk-ua.facebook.com/"
+                target="_blank"
+              >
+                <img src="./images/icons/facebook.svg" alt="facebook">
+              </a>
+
+              <a
+                class="footer__icons-link"
+                href="https://www.instagram.com/"
+                target="_blank"
+              >
+                <img src="./images/icons/instagram.svg" alt="instagram">
+              </a>
+            </div>
+          </div>
+
+          <div class="footer__content">
+            <div class="footer__body">
+              <div class="footer__shedule" id="about-us">
+                <div class="footer__subtitle">Графік роботи</div>
+
+                <div class="footer__shedule-day">
+                  <div class="footer__text">
+                    <span class="footer__time">12:00 — 20:00 </span> середа <br> <span class="footer__time">12:00 — 20:00</span> четвер <br> <span class="footer__time">12:00 — 20:00</span> п’ятниця <br> <span class="footer__time">11:00 — 19:00</span> субота <br> <span class="footer__time">11:00 — 19:00</span> неділя
+                  </div>
+
+                  <div class="footer__weekend">
+                    <span class="footer__time">Вихідні:</span> понеділок, вівторок <br>Каса припиняє роботу за 1 годину до закриття музею
+                  </div>
+                </div>
+              </div>
+
+              <div class="footer__contacts" id="contacts">
+                <div class="footer__contacts-name">Контакти:</div>
+
+                <div class="footer__adress">
+                  <a
+                    class="footer__adress-link"
+                    href="https://goo.gl/maps/RvopVkvbDjsBpsSH7"
+                    target="_blank"
+                  >вул. М. Грушевського, 6; Київ, 01001</a>
+                </div>
+
+                <div class="footer__phone">
+                  <a
+                    class="footer__phone-link"
+                    href="tel:+38 (044) 278-13-57"
+                  >тел. +38 (044) 278-13-57 <br> факс +38 (044) 278-74-54</a>
```

Comment:
Need to fix format phone only in href
```suggestion
                  <a
                    class="footer__phone-link"
                    href="tel:+380442781357"
                  >
                    тел. +38 (044) 278-13-57 <br>
                    факс +38 (044) 278-74-54
                  </a>
```